.pre-commit-config.yaml

```
@@ -0,0 +1,12 @@
+repos:
+
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.1.0
+  hooks:
+  - id: trailing-whitespace
+  - id: check-yaml
+
+- repo: https://github.com/codespell-project/codespell
+  rev: v2.1.0
+  hooks:
+  - id: codespell
```

Maybe here we could add the style tooling. But it might have some side effects that you anticipated.
```suggestion
  - id: codespell
  - repo: https://github.com/psf/black
  rev: 22.3.0
  hooks:
  - id: black
    args:
      - --line-length=120
- repo: https://github.com/asottile/reorder_python_imports
  rev: v3.0.1
  hooks:
  - id: reorder-python-imports
    args: 
    - --py37-plus
- repo: https://gitlab.com/PyCQA/flake8
  rev: 3.9.2
  hooks:
  - id: flake8
    args:
      - --max-line-length=120
```

---

As per previous comment, this will not be "seen" by other repos. Since the scope of this `.pre-commit-config.yaml` file is local and no source code is expected to be included here, there is no need to include "black", "isort" or "flake8".

---

Make sense.